src/main/java/seedu/mindmymoney/data/CreditCardList.java

```diff
@@ -0,0 +1,76 @@
+package seedu.mindmymoney.data;
+
+import seedu.mindmymoney.userfinancial.CreditCard;
+
+import java.util.ArrayList;
+
+/**
+ * Container for Credit Cards.
+ */
+public class CreditCardList {
+    public ArrayList<CreditCard> creditCardListArray;
```

Comment:
Consider making this attribute private, and exposing a getter to it, to improve encapsulation.

Reply:
oh this is referred from expenditure list!